ui/admin/app/components/form/role/manage-custom-scopes/index.js

```diff
@@ -10,52 +10,117 @@ import { tracked } from '@glimmer/tracking';
 export default class FormRoleManageCustomScopesIndexComponent extends Component {
   // =attributes
 
-  @tracked showRemoveOrgModal = false;
-  @tracked selectedOrg;
+  @tracked selectedOrgs = [];
+  @tracked selectedOrg = '';
+
+  /**
+   * Returns the display name of the selectedOrg.
+   * @type {string}
+   */
+  get orgDisplayName() {
+    const org = this.args.model.orgScopes.find(
+      ({ id }) => id === this.selectedOrg,
+    );
+    return org.displayName;
+  }
 
   // =actions
 
   /**
    * Handles the selection changes for the paginated table.
    * @param {object} selectableRowsStates
+   * @param {object} selectionKey
    */
   @action
   selectionChange({ selectableRowsStates, selectionKey }) {
     const { role, projectTotals } = this.args.model;
-    selectableRowsStates.forEach((row) => {
-      const { isSelected, selectionKey: key } = row;
-      const includesId = role.grant_scope_ids.includes(key);
-      if (isSelected && !includesId) {
-        // Add the org id if it was selected and it doesn't exist as a grant scope.
-        role.grant_scope_ids = [...role.grant_scope_ids, key];
-      } else if (!isSelected && includesId) {
-        // Remove org id if it was deselected and it does exist as a grant scope.
+
+    const addOrRemoveValues = (add, remove, orgId) => {
```

Comment:
Was it intentional that selecting orgs automatically selects all projects underneath it now? Before this change it didn't auto select any projects and you had to actually go in now.

It seems like now there's no easy way to just select only orgs?

Reply:
yeah it was intentional, originally i had asked about that change being a modal in this [slack thread](https://hashicorp.slack.com/archives/C06LR1KKN4E/p1723568423341309).

Reply:
So there's no way to just select all the orgs only?

Reply:
Nope, not on the manage custom scopes page. The only way would be to select the org save it (it auto selects projects) then go into the manage projects page and deselect all projects. Would you prefer if i move this specific change to a different PR?

Reply:
changes around this have been removed! thank u for the feedback!